damus/Views/ActionBar/EventDetailBar.swift

```diff
@@ -17,15 +17,15 @@ struct EventDetailBar: View {
             if bar.boosts > 0 {
                 Text("\(bar.boosts)")
                     .font(.body.bold())
-                Text("Reposts")
+                Text("Repost".conformPlurality(count: bar.boosts))
```

Comment:
I've already fixed this issue and it's in master:
https://github.com/damus-io/damus/pull/320/files#diff-548f04cd57f663aa1e8b4ddc8e21af44ea6ae1f2c360a0a82b8135aa5ca94fcfR1

Sorry that you spent time on this!